package.json

```diff
@@ -31,6 +31,7 @@
     "test:unit": "npm run mocha -- 'test/unit/**/*.test.ts'",
     "test:unit:cover": "nyc --report-dir coverage/unit --exclude 'src/@types/**/*' npm run test:unit",
     "test:integration": "npm run mocha -- 'test/integration/**/*.test.ts'",
+    "test:guide": "npm run mocha -- 'test/integration/CodeExamples.test.ts'",
```

Comment:
I think we can ditch this

Reply:
Ok sure, I just put it there to make it easy to test that one file